mediapipe/tasks/ios/vision/core/utils/sources/MPPImage+Utils.mm

```diff
@@ -133,19 +143,27 @@ + (uint8_t *)rgbPixelDataFromCVPixelBuffer:(CVPixelBufferRef)pixelBuffer error:(
 
   CVPixelBufferUnlockBaseAddress(pixelBuffer, 0);
 
-  return rgbPixelData;
+  if (!rgbPixelData) {
+    return nullptr;
+  }
+
+  std::unique_ptr<ImageFrame> imageFrame =
+      absl::make_unique<ImageFrame>(::mediapipe::ImageFormat::SRGB, /*width=*/width,
```

Comment:
Nit: No need for argument comments here since the arguments are self-explanatory (but you can leave as you prefer)

Reply:
I have removed all except the deleter